ansible_vars/test_vars.yaml

```diff
@@ -0,0 +1,11 @@
+---
+package_version: "test"
+node_labeller_registry: "test"
+node_labeller_version: "test"
+cpu_plugin_registry: "quay.io/kubevirt"
+cpu_plugin_version: "v0.1.0"
```

Comment:
I am a bit worried about the versions here. Who will maintain them? The whole CI test can be for nothing if those are not accurate..

Reply:
this test is only for node labeller, not for all plugins.